.github/workflows/android.yml

```diff
@@ -156,6 +160,9 @@ jobs:
           distribution: zulu
           cache: gradle
 
+      - name: Set up Android SDK and NDK for 16KB page size alignment
+        run: ./scripts/setup-android-ndk-16kb
```

Comment:
This script enables the 16KB page size alignment for Android builds as highlighted on [this comment](https://github.com/stellar/freighter-mobile/issues/383#issuecomment-3325070235).